src/server/common/helpers/extensions.js

```diff
@@ -1,7 +1,8 @@
 import { provideService } from '~/src/server/common/helpers/provide-service.js'
```

Comment:
Not needed now but I wonder if this files needs a better location? At the moment its very global being called extensions. Where as this is extensions for services and test-suites? Something to think about maybe?

Reply:
Mind you the functions kind of do that... 🤔 maybe not then

Reply:
Def feel like we need a name that encompasses services & test-suites. a lot of shared components/logic between them. I think `resources` would be too general.